apps/tutorial_servo_control/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(tutorial_servo_control.${TARGET}-board.elf)
```

Comment:
the training asks u to make 2 packages tutorial_servo_pot_control and tutorial_servo_can_control. this please follow that its easier to check ur code